source/addComment.js

```diff
@@ -1,3 +1,19 @@
 $(document).ready(function(){
-  // add your code here
+  $( '#new_comment_button' ).on('click', function() {
+    var formComment = '<textarea name="comment" id="comment" placeholder="Write your comment"></textarea></br>';
+    var formAuthor = '<input type="text" name="author" id="author" placeholder="Author" /></br>';
+    var formSubmit = '<input type="submit" value="Submit Comment" />';
+    $('#comments').append('<form id ="new_comment_form">' + formComment + formAuthor + formSubmit + '</form>');
+    $('#new_comment_form').on('submit', function(event){
```

Comment:
I would move this listener outside of the on click listener. They should be 2 separate functions